packages/ui/src/components/va-switch/VaSwitch.vue

```diff
@@ -40,7 +40,9 @@
         >
           <div
             v-if="computedInnerLabel || $slots.innerLabel"
-            class="va-switch__track-label">
+            class="va-switch__track-label"
+            :style="{ color: textColorComputed }"
```

Comment:
Think, it will be better to make an exposed computedStyles object (may be we would bind another styles for track label in future).